pyproject.toml

```diff
@@ -99,5 +99,5 @@ testpaths = [
 datasetinsights = "datasetinsights.__main__:entrypoint"
 
 [build-system]
-requires = ["poetry>=1.0.5"]
+requires = ["poetry>=1.0.10"]
```

Comment:
@Saurav-D This does not pin poetry version to 1.0.10.

Reply:
Yes. It's working without pinning poetry version.